app/screens/settings/settings.tsx

```diff
@@ -148,6 +144,27 @@ const Settings = ({componentId, showHelp, siteName}: SettingsProps) => {
         goToScreen(screen, title);
     });
 
+    const goToAdvancedSettings = preventDoubleTap(() => {
```

Comment:
0/5 depending on how often this will re-render, and how complex / reactive the child components are, it may be worth it to add some kind of useCallback in these two functions.

Reply:
no callback needed in my opinion